### PR TITLE
[DOC] Additions to include before responding to reviews of the pre-print

### PR DESCRIPTION
**Pull request by oesteban**
EDITED

- [x] Section about limitations and why not to use fmriprep
- [x] Section about transferring data to server
- [x] Revision of AROMA documentation

### CODE REVIEW

**Comment by oesteban** [comment]
Check:
- https://2131-53608443-gh.circle-artifacts.com/0/home/circleci/out/docs/index.html#limitations-and-reasons-not-to-use-fmriprep
- https://2131-53608443-gh.circle-artifacts.com/0/home/circleci/out/docs/usage.html#not-running-on-a-local-machine-data-transfer
- https://2131-53608443-gh.circle-artifacts.com/0/home/circleci/out/docs/workflows.html#ica-aroma

File: fmriprep/workflows/bold/confounds.py [hunk 1]
```diff
@@ -373,18 +374,23 @@ def init_ica_aroma_wf(template, metadata, mem_gb, omp_nthreads,
...
-        (columns named ``AROMAAggrCompXX``)
+       (columns named ``AROMAAggrCompXX``)
 
     Additionally, non-aggressive denoising is performed on the BOLD series
     resampled into MNI space.
 
+    There is a current discussion on whether other confounds should be extracted
+    before or after denoising `here <http://nbviewer.jupyter.org/github/poldracklab/\
+    fmriprep-notebooks/blob/922e436429b879271fa13e76767a6e73443e74d9/issue-817_\
+    aroma_confounds.ipynb>`__.
```
Comment (effigies) [inline fmriprep/workflows/bold/confounds.py]: A couple relevant issues to this discussion: #1049, #1136.

I'm not sure it's a good idea to include these somehow, but just want to rope those discussions in here.

Comment (oesteban) [reply fmriprep/workflows/bold/confounds.py]: Those issues are cited in the notebook @jdkent wrote (and pointed by the link). I assumed that was the best route to them as James introduce them with some context.

Comment (effigies) [reply fmriprep/workflows/bold/confounds.py]: Ah, cool. I didn't actually check the notebook. Lgtm.

**Comment by oesteban** [comment]
Okay, I'll go ahead and merge this. We can reiterate at any point if we need to come back to this.

**Comment by jdkent** [comment]
as a post hoc review, this looks good to me. Sorry for not getting back to this sooner.

**Pull request merged**